src/tests/examples/example_test.py

```diff
@@ -87,6 +94,15 @@ def check_integrity_second_opt(path):
 
     np.testing.assert_allclose(reference_values[4:8], output_values[4:8], rtol=1e-4, atol=0)
 
+def fetch_hourly_hydro_reservoir_values(path):
```

Comment:
We could have a more common function `fetch_hourly_values` that will take the column to be read as a parameter

Reply:
Done